packages/clerk-js/src/ui/elements/Navbar.tsx

```diff
@@ -118,6 +118,9 @@ export const NavBar = (props: NavBarProps) => {
 
   useEffect(() => {
     routes.every(route => {
+      if (route.external) {
+        return true;
```

Comment:
Yes, in core-2 we basically say "an external route will never be active".

in core-1 we need to return true otherwise the .every function will stop execution for the rest of the elements. In case the active id was not found this would be problematic. Basically we say "if it is external, just skip".